kolibri/plugins/device/assets/src/views/DeviceIndex.vue

```diff
@@ -181,25 +64,19 @@
         this.$store.commit('SET_WELCOME_MODAL_VISIBLE', false);
       },
     },
-    $trs: {
-      deviceManagementTitle: {
-        message: 'Device',
-        context:
-          'The device is the physical or virtual machine that has the Kolibri server installed on it.',
-      },
-      permissionsLabel: {
-        message: 'Permissions',
-        context:
-          'Indicates the Device > Permissions tab. Permissions refer to what users can manage on the device.',
-      },
-    },
   };
 
 </script>
 
 
 <style lang="scss" scoped>
 
+  /* Default styles for KPageContainer in Device */
+  /deep/ .device-container {
```

Comment:
Could we make this more explicit, at the risk of repetition, by adding to the specific uses of KPageContainer? Seems to be relying on the addition of the `device-container` class in each page anyway, so might be better to just be explicit about what that means?

Reply:
What do you think of this compromise https://github.com/learningequality/kolibri/pull/9475/commits/eb2ca57daa43d0c6374d32f1b94405ce33d72e4b

I'm keen to keep the styles living in one place - I figure we're not entirely unlikely to want to adjust all pages at once at some point. This way we get both - explicit "here is why this class is here" and "here is where that lives and is defined".

Reply:
I like it - keeps a single source of truth, while also being completely explicit within the component.

Reply:
The other advantage of using a mixin rather than just a defined class that is imported is that it allows for specific deviation within any of the page components if needed.